### PR TITLE
#34 Change status value to boolean

### LINKED ISSUE #34: Change status value to boolean
In the application, where we want to set status for either response from 3rd party API or sending an answer from our server, we use `0 - failed` and `1 - success`.

it might be better to use a boolean instead of these status codes, as true & false are more evident.

### PR DESCRIPTION
**Pull request by LogaMithran**
Boolean gives much more clarity and code readablity

Tests
{
    "status": true,
    "message": "User created successfully",
    "data": {}
}
{
    "status": false,
    "type": "fail",
    "error": {
        "index": 0,
        "code": 11000,
        "keyPattern": {
            "email": 1
        },
        "keyValue": {
            "email": "user-1@gmail.com"
        },
        "statusCode": 500,
        "status": false,
        "type": "fail"
    }
}

### CODE REVIEW

File: app/utils/imageUpload.js [hunk 1]
```diff
@@ -24,9 +24,9 @@ export const uploadProfileImage = async (file, fileName) => {
 
     const data = await uploadImage(tempFilePath, fileName, UPLOAD_PRESETS.profileImages);
 
-    return [1, data];
+    return [true, data];
   } catch (error) {
-    return [0, new AppError('Unable upload image. Please try again.', 401)];
+    return [false, new AppError('Unable upload image. Please try again.', 401)];
```
Comment (ArunBohra12) [inline app/utils/imageUpload.js:29]: @LogaMithran, can you please check here, `app/controllers/userController.js:31`

we use the following still and it isn't updated yet
```js
const data = await uploadProfileImage(req.file, fileName, next);

if (data[0] === 0) {
  return next(data[1]);
}
```

**Review by ArunBohra12 — CHANGES_REQUESTED** [review]
Can you please add this change?

File: app/utils/imageUpload.js (hunk 1 above)
Comment (LogaMithran) [reply app/utils/imageUpload.js:29]: @ArunBohra12 Now I have updated it

Comment (ArunBohra12) [reply app/utils/imageUpload.js:29]: Earlier, we wrote the condition as `data[0] === 0`, now we need to check if it is false so I added `!data[0]` the exclamation (to run condition if it is false) 😉.

**Review by ArunBohra12 — APPROVED** [review]
Looks good now 🔥🔥

**Pull request merged**